.idea/runConfigurations.xml

```diff
@@ -1,13 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this got automatically deleted when updating. I don't think we had anything custom in here anyways and it was just the default run configurations that get generated